### ISSUE TITLE
Feature/Enhancement : Deepin Editor tabs on start ...

### ISSUE THREAD

**Issue by aiamuzz** [linuxdeepin/developer-center#982]
Hi,

I like the current implementation/handling of the unsaved (Blank Document) tabs ... 

Currently we can open multiple tabs (blank documents) ... and if we close the editor application with those open tabs the unsaved tabs reopen from exactly the same point when they were closed. I love this option ... I use this as a rough doodle sheet ... and its available every time i open the editor.

**I was hoping that it would be a nice idea to include even 'saved' file tabs to open from the previous session along with the Blank document(unsaved) tab sheets(something similar to how chrome/chromium/firefox lets us open the application from where it was left from the last session) !!! 

It will be very convenient should you decide to implement it !!!**

currently opening unsaved tabs from the last session is very convenient but opening the saved text files requires us to navigate and open it from the file chooser window seems very inconvenient in comparison to the unsaved(blank document) that open currently !!!

**Comment by aiamuzz** [comment]
![image](https://user-images.githubusercontent.com/3283676/55091877-98027780-50d7-11e9-9aa5-55aa511a6e5f.png)

@hualet ... @BLumia ... can somebody tell me where(the location) these unsaved 'Blank Documents' are saved until we save it ...

**Comment by hualet** [comment]
The unsaved files are located at `~/.local/share/deepin/deepin-editor/blank-files`.

**Comment by aiamuzz** [comment]
@hualet ... you closed the thread ... but we didn't discuss the feature request / enhancement mention in the OP ... ?

**Comment by aiamuzz** [comment]
The feature i am requesting is when i open the deepin-editor i want my most used text files to open along with the blank-files ...

this will enhance a user experience in the same way a browser open from the point 'Continue from where you left off' ... **in simpler terms ... a way to add some favorite text files to open on deepin-editor startup ...**

**Comment by hualet** [comment]
Oh, you mean session restoration.

**Comment by tsic404** [comment]
Now deepin-editor will automaticaly open last tabs.
